docs/modules/ROOT/pages/cli_reference.adoc

```diff
@@ -5,6 +5,11 @@ In this section you will learn about:
 * [x] ways to configure `ike` (through flags, files et al)
 * [x] complete list of commands and their available flags
 
+[#completion]
+== Autocomplete
+
+If you are using `zsh` or `bash` you can easily enable autocomplete for `ike` by invoking `. <(ike completion SHELL)`.
+
```

Comment:
Should this not have a section like:

   include::cmd:ike[flags='completion --help --help-format=adoc']

Reply:
Not sure if that would be better

Reply:
I mean this is one-off not changing stuff, but in general, I'm thinking of extending help generation to have not only flags output, but also other bits in the template, include usage and example if exists.

On top of that iterating over all the commands instead of having section added manually.

I've opened issue for that #182. Happy to work on it next.

Reply:
Didn't mean instead of. Just thinking if we explain "all" commands in one place in one way we might as well explain all.

Reply:
We don't yet - we just print flags.

Reply:
Oh sure, I just meant under "== Available commands"

Reply:
#182